src/mockSdk.js

```diff
@@ -17,6 +17,18 @@ export const mockSDKPerson = {
   orgId: 'orgID',
 };
 
+export const mockSDKPersonList = {
+  items: [{
+    id: 'personIDCurrentUser',
```

Comment:
`id: 'personID'` should be sufficient, no need for a "current user" concept here